### DISCUSSION TITLE
2sxc 13.10 LTS Released - for Oqtane 3.1

### DISCUSSION THREAD

**Discussion by iJungleboy** [oqtane/oqtane.framework#2173] [Show and tell]
Just released 2sxc 13.10 LTS - these are our most stable releases. 

It adds content-management and Dynamic Data Models, EAV, etc. to Oqtane just like on Dnn. 

Best read the blog: https://2sxc.org/en/blog/post/2sxc-13-10-lts-may-the-4th-be-with-you

💘 from Switzerland, 
iJungleboy

**Comment by qunshen** [comment]
do you have a working Oqtane Based demo app that utiliize 2sxc?
